lib/constants.go

```diff
@@ -101,7 +101,24 @@ const (
 	//
 	// At the time of this writing, the intent is to deploy it in a backwards-compatible
 	// fashion, with the eventual goal of phasing out blocks with the previous version.
-	HeaderVersion1       = uint32(1)
+	HeaderVersion1 = uint32(1)
```

Comment:
should we convert these HeaderVersions to some type now?

Reply:
Sure, it's a simple enough change to do update the type here. I created a new type `MsgDeSoHeaderVersion` and set it as an alias to uint32 so everything compiles.